manifests/server/puppetserver.pp

```diff
@@ -252,22 +252,90 @@
     content => template('puppet/server/puppetserver/conf.d/puppetserver.conf.erb'),
   }
 
-  file { "${server_puppetserver_dir}/conf.d/webserver.conf":
+  file { "${server_puppetserver_dir}/conf.d/auth.conf":
```

Comment:
It worked until now the way it was, because the files came with the puppetserver package, but we can add that (later ;)).